pkg/k8s/waiter.go

```diff
@@ -103,16 +153,8 @@ func (w *Waiter) Wait(timeout time.Duration) bool {
 			}
 			if !printed[r.QName()] {
 				var name string
-				objIf, ok := r["involvedObject"]
-				if ok {
-					obj, ok := objIf.(map[string]interface{})
-					if ok {
-						name = fmt.Sprintf("%s/%v.%v", obj["kind"], obj["name"],
-							obj["namespace"])
-						name = watcher.Canonical(name)
-					} else {
-						name = r.QName()
-					}
+				if obj, ok := r["involvedObject"].(map[string]interface{}); ok {
+					name = w.canonical(fmt.Sprintf("%s/%v.%v", obj["kind"], obj["name"], obj["namespace"]))
```

Comment:
I feel like that Sprintf needs to be modified to include a GV, and not just `obj["kind"]`.